src/checkstyle/checkstyle-suppressions.xml

```diff
@@ -6,4 +6,6 @@
 	<suppress files=".+Application\.java" checks="HideUtilityClassConstructor"/>
 	<suppress files="[\\/]initializr-service-sample[\\/]" checks="JavadocType"/>
 	<suppress files="[\\/]initializr-service-sample[\\/]" checks="ImportControl"/>
+	<suppress files="initializr-generator-spring/src/test/java/io/spring/initializr/generator/spring/container/dockercompose/DockerComposeServiceTests.java" checks="SpringLeadingWhitespace"/>
```

Comment:
Is YAML with tabs legit? If so we could configure the indent and use tabs in test. That's what I've done for code so far.

Reply:
Unfortunately not:

```
yaml: line 2: found character that cannot start any token
```

I could remove the multiline strings and extract the files into resources and read them in the tests to get rid of the exclusions.